src/styles/utils/_variables.scss

```diff
@@ -0,0 +1,25 @@
+$stopwatch-2r: 500px;
+$stopwatch-r: $stopwatch-2r * 0.5;
+$center-2r: $stopwatch-r * 0.06;
+$seconds-2r: $stopwatch-r * 0.4;
```

Comment:
better to give the variables more clear names

Reply:
I went with the mathematical terminology. I guess I could've just used 'radius' for more clarity. I will have that in mind for future projects. Thanks 👍